src/utils/auth/sanitizationUtils.ts

```
@@ -63,11 +62,15 @@ export function sanitizeInput(input: string | null, options: SanitizationOptions
     result = result.substring(0, options.maxLength);
   }
 
-  // Use DOMPurify for additional sanitization
-  return DOMPurify.sanitize(result, {
-    ALLOWED_TAGS: [],
-    ALLOWED_ATTR: [],
-  }).trim();
+  // Additional comprehensive sanitization without DOM dependencies
+  result = result
+    .replace(/[\0\x08\x0B\x0C\x0E-\x1F]/g, '') // Remove control characters
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)